tests/components/yale/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""Tests for the yale component."""
+
+MOCK_CONFIG_ENTRY_DATA = {
```

Comment:
Looks like something for const

Reply:
stale, removed in 62ed0c1bac92a671575eba99419b1ff07e4a59e0